examples/images/diffusion/configs/train_colossalai_teyvat.yaml

```diff
@@ -1,7 +1,8 @@
 model:
-  base_learning_rate: 1.0e-04
+  base_learning_rate: 1.0e-4
   target: ldm.models.diffusion.ddpm.LatentDiffusion
   params:
+    parameterization: "v"
     linear_start: 0.00085
```

Comment:
Add a comment for the meaning of `parameterization: v`

Reply:
![image](https://user-images.githubusercontent.com/55798671/207204569-b2f327ae-0880-4875-986d-ca2e2cda0a63.png)

Reply:
It means how to define the loss

Reply:
I wonder if there is any doc to explain the parameters, which can significantly help the users to understand the config file.